app/controllers/spotlight/filters_controller.rb

```diff
@@ -11,14 +11,14 @@ def create
       else
         flash[:alert] = @filter.errors.full_messages.join('<br/>'.html_safe)
       end
-      redirect_to spotlight.edit_exhibit_path @exhibit, anchor: 'filter'
+      redirect_to spotlight.edit_exhibit_path @exhibit, params: { tab: 'filter' }
```

Comment:
I'm kind of surprised to the `params` key.  I wasn't aware that `params` was a special thing (I don't see any reference to it in the `redirect_to` documentation.  Does this offer any benefit over doing something like

```ruby
redirect_to spotlight.edit_exhibit_path(@exhibit, tab: 'filter')
```
?

Reply:
Nope! No extra benefits. Looks like I might have been referencing an outdated snippet... I've updated this PR with the more concise syntax. ✂️